src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+
+body {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 96px;
+  height: 116px;
+  margin: auto;
+}
+
+.stars {
+ display: flex;
+ flex-direction: row;
```

Comment:
row is a default value in flex-direction, if you remove 'flex-direction: row ' nothing will changed